odoo/addons/base/data/res_users_data.xml

```diff
@@ -20,8 +20,7 @@ System</span>]]></field>
             <field name="company_ids" eval="[Command.link(ref('main_company'))]"/>
             <field name="groups_id" eval="[Command.set([])]"/>
             <field name="image_1920" type="base64" file="base/static/img/avatar_grey.png"/>
-            <field name="signature"><![CDATA[<span>-- <br/>
-Administrator</span>]]></field>
+            <field name="signature" type="xml"><span>-- <br/>Mitchell Admin</span></field>
```

Comment:
You are modifying user data, not demo data.
Every time somebody set up a database without demo data, his signature will be Mitchell Admin. It is quite a worst user experience. Try to set up a database without demo data, change the admin name and see how you can avoid to sign for Mitchell Admin or "administrator". Then you need to modify the demo data to override the "Administrator" signature.

Reply:
@arj-odoo , done the changes. Thanks